src/book/02-projects/01-nubots/03-build-system.mdx

```diff
@@ -250,7 +250,10 @@ To run roles locally on your system use the `run` command of `b` as follows
 ---
 
 **NOTE**
-Be sure that you are using the `generic` target for this (`./b target generic`), otherwise you role will likely crash.
+Be sure that you are using the `generic` target for this (`./b target generic`), otherwise your role will likely crash (unless your CPU happens to be the same or newer than the 7th generation Core i7 that the robots are current using).
+
+---
```

Comment:
What are these `---`s for

Reply:
They draw horiztonal lines. They will be removed once #23 gets merged in